submissions/yaripey/dom-task/js/main.js

```diff
@@ -45,33 +45,23 @@ const texts = {
 document.addEventListener('DOMContentLoaded', () => {
   const toggler = document.querySelector('.toggle')
 
-  // Function for changing text in main
   const changeText = (target) => {
     const currentMain = document.querySelector('main');
 
-    if (currentMain.classList[0] !== target.id) { // if this page isn't open already
```

Comment:
Why did you remove this check? Please see my previous comment and make this logic work.

Reply:
I removed the logic and shifted some responsibilities in functions, now page rerenders only if user hits a button that is different than what is focused right now.